src/vs/editor/common/editorContextKeys.ts

```diff
@@ -16,6 +16,11 @@ export namespace EditorContextKeys {
 	 */
 	export const focus = new RawContextKey<boolean>('editorFocus', false);
 
+	/**
+	 * A context key that is set when any editor input has focus (regular editor, repl input...).
+	 */
+	export const inputFocus = new RawContextKey<boolean>('inputFocus', false);
```

Comment:
Let's ask in the stand-up what would be a good name. Perhaps `textInputFocus` is better than `inputFocus`, as there might in the future be other types of `input`s, like drop downs or radio buttons...